vic/drivers/image/include/vic_driver_image.h

```diff
@@ -187,6 +203,7 @@ void vic_start(void);
 void vic_store(void);
 void vic_write(void);
 char will_it_snow(double *t, double t_offset, double max_snow_temp,
-                  double *prcp, size_t n);
+                  double *prcp,
+                  size_t n);
```

Comment:
are all of these whitespace changes from uncrustify?  They don't seem necessary.  Were you using the uncrustify configuration file in the VIC repo?